kt/entry-generation/godot-entry-generator/src/main/kotlin/godot/entrygenerator/generator/hintstring/PropertyHintStringGenerator.kt

```diff
@@ -0,0 +1,15 @@
+package godot.entrygenerator.generator.hintstring
+
+import godot.entrygenerator.model.PropertyHintAnnotation
+import godot.entrygenerator.model.RegisteredProperty
+
+abstract class PropertyHintStringGenerator(
+    val registeredProperty: RegisteredProperty
+) {
+    protected val propertyHintAnnotation = registeredProperty.annotations.firstOrNull { it is PropertyHintAnnotation } as PropertyHintAnnotation?
```

Comment:
Are we sure there is no case of several HintAnnotations needed?

Reply:
Yes. The code in the inspector which handles hint strings (https://github.com/godotengine/godot/blob/4fcc5891450a7dca9c8ebae6ff3765f5f060c442/editor/editor_properties.cpp#L2701) only handles one and this one according to the property hint: https://github.com/godotengine/godot/blob/dcd11faad3802679a43b27155f1b6bc59aa39b60/core/object.h#L56.